PrototopeJSBridge/GestureBridge.swift

```diff
@@ -400,6 +431,20 @@ public class ContinuousGesturePhaseBridge: NSObject, BridgeType {
 	public class func addToContext(context: JSContext) {
 		context.setObject(self, forKeyedSubscript: "RotationGesture")
 	}
+    
+    public var shouldRecognizeSimultaneouslyWithGesture: JSValue? {
```

Comment:
Looks like we have an opportunity to extract these implementations to a superclass, too, but can happen in a future revision.